src/components/page-heading/_page-heading.scss

```diff
@@ -1,18 +1,21 @@
 .hmrc-page-heading {
   display: table;
   width: 100%;
-  margin-top: govuk-em(60, 32);
-  margin-bottom: govuk-em(30, 32);
   padding-top: 8px;
   clear: both;
   table-layout: fixed;
 
+  [class*="govuk-caption"] {
```

Comment:
Is there a reason we need to swap the class selector with an attribute selector? This raises the specificity value which is generally not a good thing. I'd prefer if we kept with BEM principles to keep specificity as low as possible. Could we keep the `.hmrc-caption-xl` – maybe rename to `.hmrc-caption` – and add `.hmrc-caption` and `.govuk-caption-xl` to the nunjucks.

Reply:
I have updated the styling in accordance with your comment on specificity and in keeping with BEM principles.